docker.d/worker.yml

```diff
@@ -1,147 +1,147 @@
 work_dir: { "$eval": "WORK_DIR" }
-artifact_dir: { "$eval": "ARTIFACTS_DIR" }
 verbose: { "$eval": "VERBOSE == 'true'" }
 jarsigner_key_store: { "$eval": "JARSIGNER_KEY_STORE" }
 do_not_contact_google_play: { "$eval": "ENV != 'prod'" }
 taskcluster_scope_prefixes:
   $if: 'COT_PRODUCT == "mobile"'
   then:
-    - project:mobile:focus:releng:googleplay:product:
-    - project:mobile:reference-browser:releng:googleplay:product:
-    - project:mobile:fenix:releng:googleplay:product:
+    - "project:mobile:focus:releng:googleplay:product:"
+    - "project:mobile:reference-browser:releng:googleplay:product:"
+    - "project:mobile:fenix:releng:googleplay:product:"
   else:
-    - project:releng:googleplay:
+    - "project:releng:googleplay:"
 products:
-  $match:
-    'COT_PRODUCT == "firefox" && ENV == "prod"':
-      - product_names: [ "beta", "aurora", "release" ]
-        digest_algorithm: SHA1
-        use_scope_for_channel: true
-        apps:
-          release:
-            google_play_track: production
-            service_account: { "$eval": "GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_RELEASE" }
-            package_names:
-              - "org.mozilla.firefox"
-            google_credentials_file: { "$eval": "GOOGLE_CREDENTIALS_FIREFOX_RELEASE_PATH" }
-            certificate_alias: release
-          beta:
-            google_play_track: production
-            service_account: { "$eval": "GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_BETA" }
+  $flatten:
+    $match:
+      'COT_PRODUCT == "firefox" && ENV == "prod"':
+        - product_names: [ "beta", "aurora", "release" ]
+          digest_algorithm: SHA1
+          use_scope_for_channel: true
+          apps:
+            release:
+              google_play_track: production
+              service_account: { "$eval": "GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_RELEASE" }
```

Comment:
We can hard-code the `service_account` values, rather than securely environment-variable-ing them in, since they're just like usernames.
I'd recommend still keeping `google_credentials_file` as-is, though

Reply:
Good point. I'll file a bug to address this.